models/classes/runner/config/QtiRunnerConfig.php

```diff
@@ -212,4 +224,33 @@ protected function getCategories(RunnerServiceContext $context)
     {
         return $context->getCurrentAssessmentItemRef()->getCategories()->getArrayCopy();
     }
+
+    private function isThemeSwitcherEnabled()
+    {
+        $config = \common_ext_ExtensionsManager::singleton()->getExtensionById('taoTests')->getConfig('test_runner_plugin_registry');
+
+        if (array_key_exists(self::TOOL_ITEM_THEME_SWITCHER_KEY, $config) && $config[self::TOOL_ITEM_THEME_SWITCHER_KEY]['active']) {
+            return true;
+        }
+    }
+
+    private function guessTestTheme()
+    {
+        $request = \Context::getInstance()->getRequest();
```

Comment:
You should never access a request (controller stuff) within a service. 
A service needs to be stateless and I/O oriented.

Reply:
Sorry I didn't see the `buildConfig` doesn't have access to the session/context.
 -  use the `options` object to send the active theme namespace, see https://github.com/oat-sa/extension-tao-testqti/pull/1330/files#diff-b0de81bf25f20349c0102da237a83ac7L160

Reply:
And please check if the backend unit tests needs to be update also